spark/src/main/java/org/opensearch/sql/spark/data/constants/SparkConstants.java

```diff
@@ -101,7 +102,7 @@ public class SparkConstants {
   public static final String ICEBERG_SPARK_EXTENSION =
       "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions";
   public static final String ICEBERG_SPARK_RUNTIME_PACKAGE =
-      "org.apache.iceberg:iceberg-spark-runtime-3.3_2.12:1.5.0";
+      "/usr/share/aws/iceberg/lib/iceberg-spark3-runtime.jar";
```

Comment:
If EMR team changes this location, we would need a deployment in all the OS versions from 2.15.  Hopefully emr keeps backward compatibility in mind.